src/lib/graphqlErrorHandler.ts

```diff
@@ -108,11 +109,18 @@ export const formattedGraphQLError = (
   }
 
   const httpStatusCodes: number[] = []
-  ;(flattenedErrors || flattenErrors(topLevelError)).forEach(
-    e =>
+  ;(flattenedErrors || flattenErrors(topLevelError)).forEach(e => {
+    // Check for server-side errors during stitching downstream
+    // TODO: Get typing working for `ServerError`
+    // https://github.com/apollographql/apollo-link/blob/480df382cf7db486ae76c56ac2522134d77e36fa/packages/apollo-link-http-common/src/index.ts#L15
+    const stitchedStatusCode = get(e, "originalError.statusCode")
+    if (stitchedStatusCode) {
+      httpStatusCodes.push(stitchedStatusCode)
+    } else {
       e.originalError instanceof HTTPError &&
-      httpStatusCodes.push(e.originalError.statusCode)
-  )
+        httpStatusCodes.push(e.originalError.statusCode)
+    }
+  })
```

Comment:
It strikes me that both branches end-up doing the same thing, so perhaps that could be combined?